include/swift/AST/DiagnosticsParse.def

```diff
@@ -132,6 +132,8 @@ NOTE(lex_comment_start,none,
 
 ERROR(lex_unterminated_string,none,
       "unterminated string literal", ())
+NOTE(lex_unterminated_string_fix_it,none,
+      "add end quotes to string literal", ())
```

Comment:
Looks like this note is unused.

Reply:
Oh yeah I was going to implement that aswell but maybe at a later date tbh, I’ll remove it